installation-wizard/container-storage-modules/values.yaml

```diff
@@ -276,10 +276,10 @@ csi-isilon:
 ########################
 csi-vxflexos:
   enabled: false
-  version: v2.10.0
+  version: v2.10.2
```

Comment:
Here, `version` should be 2.10.1 since this is in sync with the driver version not the helm chart version for csi-vxflexos.

Reply:
fixed.